internal/testblobs/slow.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package testblobs
+
+import (
+	"context"
+	"time"
+
+	"storj.io/storj/storage"
+)
+
+// SlowBlobs implements a slow blob store.
+type SlowBlobs struct {
+	blobs storage.Blobs
+	delay time.Duration
+}
+
+// NewSlowBlobs creates a new slow blob store wrapping the provided blobs.
+// All operations are delayed with the provided duration.
+func NewSlowBlobs(blobs storage.Blobs, delay time.Duration) *SlowBlobs {
+	return &SlowBlobs{
+		blobs: blobs,
+		delay: delay,
+	}
+}
+
+// Create creates a new blob that can be written
+// optionally takes a size argument for performance improvements, -1 is unknown size
+func (b *SlowBlobs) Create(ctx context.Context, ref storage.BlobRef, size int64) (storage.BlobWriter, error) {
```

Comment:
According to our current code conventions, the `b` isn't a good name for a receiver.

Reply:
True. Renamed it.